recipes/pymks/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "pymks" %}
+{% set version = "0.3.4a3" %}
+{% set sha256 = "627083053ca4d4e0cf1ddabdbaadf931e809e19ed18dfcf9b643fb984fd90ab7" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://files.pythonhosted.org/packages/37/8a/055554e151a2abcbc0eed64e08e13965930795bf2693e74df6e4d2735fe8/{{ name }}-{{ version }}.tar.gz
```

Comment:
Just a note that you might want to update the examples to use the new PyPI URL scheme, see http://nixos.org/nix-dev/2016-April/020237.html.